langgraph/src/checkpoint/base.ts

```diff
@@ -65,14 +83,67 @@ export const enum CheckpointAt {
   END_OF_RUN = "end_of_run",
 }
 
+export interface CheckpointTuple {
+  config: RunnableConfig;
+  checkpoint: Checkpoint;
+  parentConfig?: RunnableConfig;
+}
+
+const CheckpointThreadId: ConfigurableFieldSpec = {
+  id: "threadId",
```

Comment:
@nfcampos, general question: The JS implementation serializes `threadId` and `threadTs` in `camelCase` (as opposed to `snake_case` in Python). Is there any value in making them both one way or the other?

Hypothetical scenario: Python graph checkpoints state, pauses execution, and JS graph (or some other runtime) resumes execution.

Reply:
yep would be nice to use same keys

Reply:
I'll make the changes in this PR.